sqlengine/postgres_engine.go

```diff
@@ -1,43 +1,55 @@
 package sqlengine
 
 import (
+	"bytes"
 	"database/sql"
 	"fmt"
+	"text/template"
 
 	"github.com/lib/pq" // PostgreSQL Driver
 
 	"code.cloudfoundry.org/lager"
 )
 
-const defaultStateDBName = "broker_state"
+var ensureTriggerPattern = `
```

Comment:
There's still global now, but they've moved closer to the functions that use them. I think that's fine for the reason Alex describes.